src/user_camera.js

```diff
@@ -110,20 +125,33 @@ class UserCamera extends gltfCamera
         return this.position;
     }
 
-    fitZoomToExtends(min, max)
+    fitZoomToExtents(min, max)
     {
         const maxAxisLength = Math.max(max[0] - min[0], max[1] - min[1]);
         this.zoom = this.getFittingZoom(maxAxisLength);
     }
 
-    fitCameraTargetToExtends(min, max)
+    fitCameraTargetToExtents(min, max)
     {
         for (const i of [0, 1, 2])
         {
             this.target[i] = (max[i] + min[i]) / 2;
         }
     }
 
+    fitCameraPlanesToExtents(min, max)
+    {
+        const longestDistance = vec3.distance(min, max);
+        let zNear = this.zoom - (longestDistance * 0.6);
+        let zFar = this.zoom + (longestDistance * 0.6);
+
+        zNear = Math.max(zNear, 0.0); // should not be negative
```

Comment:
set it to an epsilon, as znear should be positive

Reply:
This should be considered in Gestaltor, too